pkg/app/pipedv1/cmd/piped/piped.go

```diff
@@ -168,13 +168,13 @@ func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
 	registry := registerMetrics(cfg.PipedID, cfg.ProjectID, p.launcherVersion)
 
 	// // Configure SSH config if needed.
-	// if cfg.Git.ShouldConfigureSSHConfig() {
-	// 	if err := git.AddSSHConfig(cfg.Git); err != nil {
-	// 		input.Logger.Error("failed to configure ssh-config", zap.Error(err))
-	// 		return err
-	// 	}
-	// 	input.Logger.Info("successfully configured ssh-config")
-	// }
+	if cfg.Git.ShouldConfigureSSHConfig() {
+		if _, err := git.AddSSHConfig(cfg.Git); err != nil {
+			input.Logger.Error("failed to configure ssh-config", zap.Error(err))
+			return err
+		}
+		input.Logger.Info("successfully configured ssh-config")
+	}
```

Comment:
Should update to remove ssh key file as we do for pipedv0
ref: https://github.com/pipe-cd/pipecd/blob/341526906dfbb3b93f6feff52ccee3157cc1eb3d/pkg/app/piped/cmd/piped/piped.go#L180-L189

Reply:
Thank you, will fix it.
Note: It is supported in the PR https://github.com/pipe-cd/pipecd/pull/5769

Reply:
fixed on [01304fe](https://github.com/pipe-cd/pipecd/pull/6201/commits/01304fe667549747b17d187f670cf7914140bf96)